src/com/facebook/buck/cxx/CxxLinkableEnhancer.java

```diff
@@ -163,6 +163,14 @@ public static CxxLink createCxxLinkableBuildRule(
     // the soname.
     if (linkType == Linker.LinkType.SHARED) {
       argsBuilder.add(new StringArg("-shared"));
+
```

Comment:
won't this make all C++ libraries (even those built, say, for Android) reference that directory at runtime?